uint8 mode
uint8 NONE=0
uint8 RESET=1
uint8 EXECUTE=2
uint8 EXECUTE_FROM_START=3
string[] joint_name
float64[] start_position
float64[] target_position
float64[] expected_result_position
duration max_execution_time
---
control_msgs/JointTrajectoryControllerState[] trajectory
